test/commands/new.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from '@oclif/test';
+import inquirer from 'inquirer';
+
+describe('new', () => {
+  test
+    .stderr()
+    .stub(inquirer, 'prompt', () => {
+      return Promise.resolve({
+        filename: 'specification.yaml',
+        template: 'default-example.yaml',
+        studio: true,
+        port: 3000
+      });
+    })
```

Comment:
Remove this because we don't need to stub anything as we are testing in `--no-tty` mode. 
```suggestion

```